src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -628,8 +628,7 @@ private string GetVersionUsingHierarchicalRules(string detectedVersion)
 
         private bool IsCondaEnvironment(PythonPlatformDetectorResult pythonPlatformDetectorResult)
         {
-            if ((pythonPlatformDetectorResult.HasCondaEnvironmentYmlFile ||
-                pythonPlatformDetectorResult.HasJupyterNotebookFiles)
+            if (pythonPlatformDetectorResult.HasCondaEnvironmentYmlFile 
```

Comment:
jupiternotebook file should not be an indicator of detecting a conda app .. environment.yml will be